src/mon/NVMeofGwMap.h

```diff
@@ -102,29 +107,31 @@ class NVMeofGwMap
     const NvmeGroupKey& group_key);
 
 public:
+
```

Comment:
This is a stray newline.  Please check every line in this commit for similar accidental additions.